backend/protocol_rpc/endpoints.py

```
@@ -437,17 +437,125 @@ async def gen_call(
+        receipt = await _gen_call_with_validator(
+            session,
+            accounts_manager,
+            msg_handler,
+            transactions_parser,
+            snapshot,
+            params,
+        )
+        return eth_utils.hexadecimal.encode_hex(receipt.result[1:])[2:]
+
+
+async def sim_call(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)